dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -94,15 +94,6 @@ public static void prepare() throws Exception {
         contentletAPI = APILocator.getContentletAPI();
         hostAPI = APILocator.getHostAPI();
 
-        final List<Host> allHosts = hostAPI.findAll(systemUser, false);
-        for (final Host host : allHosts) {
-            if (host.isSystemHost() || host.getHostname().startsWith("demo")) {
-                continue;
-            }
-            hostAPI.archive(host, systemUser, false);
-            hostAPI.delete(host, systemUser, false);
-        }
```

Comment:
and is not this code need for this class's test?

Reply:
no, the tests run fine without this code.

Reply:
so, what was it used for? I have the same doubt

Reply:
I had added this code to clean up my environment locally since I run many times my test I end up with a bunch of hosts locally and it is just annoying.. I added that code and forgot to remove it from the final PR that made into master. I apologize if because of my mistake you wasted your time trying to find the cause. I'll be more careful next time.